app/assets/stylesheets/ideas.scss

```diff
@@ -1,3 +1,23 @@
-// Place all the styles related to the Ideas controller here.
-// They will automatically be included in application.css.
-// You can use Sass (SCSS) here: http://sass-lang.com/
+.ideas.listing {
+  margin-top:20px;
+}
+
+.favorite-icon {
+  &:hover {
+    cursor: pointer;
+
+    polygon {
+      stroke: #f3c967;
+    }
+  }
+}
+
+.new-idea-form {
+  background: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.